Week-5/Assignment5.4/Assig5.test.js

```diff
@@ -0,0 +1,32 @@
+const mathOperations = {
+    addition: function (a, b) {
+        return a + b;
+    },
+
+    subtraction: function (a, b) {
+        return a - b;
+    },
+    multiplication: function (a, b) {
+        return a * b
+    }
+}
+
+describe("Calculator tests", () => {
```

Comment:
There should be only one describe, and it should contain all test cases.
Use it() instead of test()

Reply:
Changed